### PR TITLE
[MOBILE-1271] Add data collection flag

### PR DESCRIPTION
**Pull request by oristanovic**
Adds data collection and push token registration flags to react native.
Manually tested in the sample app.

### CODE REVIEW

File: ios/UARCTModule/UrbanAirshipReactModule.m [hunk 1]
```diff
@@ -87,6 +95,20 @@ - (RCTBridge *)bridge {
...
+    resolve(@([UAirship shared].isDataCollectionEnabled));
+}
+
+RCT_REMAP_METHOD(isPushTokenRegistrationEnabled,
+                 isPushTokenRegistrationEnabled_resolver:(RCTPromiseResolveBlock)resolve
+                 rejecter:(RCTPromiseRejectBlock)reject) {
+
+    resolve(@([UAirship push].pushTokenRegistrationEnabled));
+}
+
```
Comment (BrianBatchelder) [inline ios/UARCTModule/UrbanAirshipReactModule.m]: Doesn't iOS need `setDataCollectionEnabled()` and `setPushTokenRegistrationEnabled()`?

Comment (rlepinski) [reply ios/UARCTModule/UrbanAirshipReactModule.m]: Its like 67 and 71

Comment (oristanovic) [reply ios/UARCTModule/UrbanAirshipReactModule.m]: @BrianBatchelder It's like the existing implementation 
```
RCT_EXPORT_METHOD(setBadgeNumber:(NSInteger)badgeNumber) {
    [[UAirship push] setBadgeNumber:badgeNumber];
}
```

Comment (BrianBatchelder) [reply ios/UARCTModule/UrbanAirshipReactModule.m]: OK, weird. I must have missed those two methods. Sorry about that!

**Pull request merged**